<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="logo_argentum">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label12">
        <property name="text">
         <string>Grupo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="logo_hexa">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ingreso mensual: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ingreso_mensual">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Porcentaje cotización: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="porcentaje_cotizacion">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Aporte mensual:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aporte">
        <property name="text">
         <string> 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Fondo elegido:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fondo_elegido">
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Edad actual:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edad_actual">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Edad jubilación: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edad_jubilacion">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Esperanza de vida: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="esperanza_de_vida">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Años de pensión:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="years_pension">
        <property name="text">
         <string> 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rango pensión mensual:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="rango_pension">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string> </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
